config/AcesContainer.pc.in

```diff
@@ -45,9 +45,9 @@
 # A.M.P.A.S., WHETHER DISCLOSED OR UNDISCLOSED.
 
 prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@INSTALL_LIB_DIR@
-includedir=@INSTALL_INCLUDE_DIR@
-AcesContainer_includedir=@INSTALL_INCLUDE_DIR@/aces
+libdir=@CMAKE_INSTALL_LIBDIR@
+includedir=@CMAKE_INSTALL_INCLUDEDIR@
+AcesContainer_includedir=@CMAKE_INSTALL_INCLUDEDIR@/aces
```

Comment:
These should be `${prefix}/@CMAKE_INSTALL_<dir>@` otherwise you get only "lib" or "include": https://people.freedesktop.org/~dbn/pkg-config-guide.html

Reply:
Thanks for review.
Sure, I can add a prefix.

Reply:
Well, I might have miss-understood. (there is already a prefix ).

Point is cmake doesn't pass variable relative to other variable. So if one ever one choose to use a custom layout, it should still work.

Reply:
Hm, not sure there is...

The current template `libdir=@CMAKE_INSTALL_LIBDIR@` will evaluate to just `libdir=lib`, no?

It needs to be `libdir=${exec_prefix}/lib` (and same for the others), as shown in the pkgconf docs linked.